gobblin-metrics-libs/gobblin-metrics/build.gradle

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied.
+
+apply plugin: 'java'
+
+/** TODO: Re-enable avro auto-compile once Java 1.7 is fully supported by users.
```

Comment:
@ibuenros I have updated the PR with an automatic generation of the java code. This has caused regeneration of the Java classes due to Avro 1.8.1 but I don't see any material change.